azure-pipelines.yml

```diff
@@ -3,111 +3,69 @@
 # Add steps that test, save build artifacts, deploy, and more:
 # https://docs.microsoft.com/azure/devops/pipelines/languages/go
 
-trigger:
-- master
-
-pool:
-  vmImage: 'Ubuntu-16.04'
+resources:
+  repositories:
+    - repository: riff
```

Comment:
riff is not designed to be nor supported as a source of ci components. This linking may (will) break at any moment.

Reply:
done!